### ISSUE TITLE
Use new child_spec

### ISSUE THREAD

**Issue by kipcole9** [whitfin/eternal#1]
Compiling on recent Elixirs produces:
```elixir
==> eternal
Compiling 5 files (.ex)
warning: Supervisor.Spec.supervise/2 is deprecated. Use the new child specifications outlined in the Supervisor module instead
  lib/eternal/supervisor.ex:Eternal.Supervisor.init/1
```

Thanks for this lib BTW, very helpful.

**Comment by jtrees** [comment]
I'm also interested in this. Has the project been abandoned?

**Comment by whitfin** [comment]
This is resolved in v1.2.2 :)

**Comment by jtrees** [comment]
Sweet, thanks!
